### PR TITLE
Add summary describing CSV row format

### PR DESCRIPTION
**Pull request by rbruggem**
Add a more human consumable description outlining what's required to
populate a CSV row.

Add 'Reference' as header for the table.

### CODE REVIEW

File: docs/guides/logistics-sheets.md [hunk 1]
```diff
@@ -110,6 +110,63 @@ Files up to 2MB are supported
 </div>
 <div>
 
+### What to input?
+
+Required inputs are:
+
+1. Route or distance
+2. Cargo weight
```
Comment (jstasiak) [inline docs/guides/logistics-sheets.md:118]: nitpick: If kilograms/tonnes then I'd call it mass, not weight, but this may be over-correct

Comment (jstasiak) [reply docs/guides/logistics-sheets.md:118]: Also: I think we need mass or containers or both; not necessarily always mass?

File: docs/guides/logistics-sheets.md [hunk 2]
```diff
@@ -110,6 +110,63 @@ Files up to 2MB are supported
...
+
+Input the distance in km in the column for `leg1_distance_km`. If you have distance data, you can use it *instead of* Route.
+
+<br />
+
+#### Cargo weight
+
+Input either cargo weight in kg or in TEU, or both.
+
+- KG: `mass_kg`
```
Comment (jstasiak) [inline docs/guides/logistics-sheets.md:149]: `KG` -> `kg`?

**Review by jstasiak — APPROVED** [review]
Nice

File: docs/guides/logistics-sheets.md (hunk 1 above)
Comment (rbruggem) [reply docs/guides/logistics-sheets.md:118]: yes, it's all explained in the Reference. 
Trying to be simpler and not as detailed.

**Pull request merged**